parser_test.go

```diff
@@ -272,12 +272,13 @@ func (s *Zuite) TestParser_parseLiteral() {
 
 func (s *Zuite) TestParser_parseType() {
 	cases := map[string]Type{
-		`undefined`: &UndefinedType{},
-		`text`:      &TextType{},
-		`bool`:      &BoolType{},
-		`number[5]`: &NumberType{5},
-		`[]bool`:    &SliceType{&BoolType{}},
-		`foobar`:    &Definition{name: "foobar"},
+		`undefined`:  &UndefinedType{},
+		`text`:       &TextType{},
+		`bool`:       &BoolType{},
+		`number[5]`:  &NumberType{5},
+		`number[32]`: &NumberType{32},
+		`[]bool`:     &SliceType{&BoolType{}},
```

Comment:
Do we have simple (non fuzz) tests for other types of slices? If not, might be nice to add those while we're in here. (Not directly related to your PR, but occurred to me from looking through the tests recently.)

Reply:
We don't! We could definitely beef up `TestParser_parseType` to have more examples like `[]number[4]` or `[][]foo`, etc. Will add to my TODO list #73.